docs/contributing/reporting-bugs.md

```diff
@@ -91,8 +91,8 @@ Steps and best practices for starting a conversation:
    have an account.
 2. Pick an **appropriate stream** to report your issue:
    - [#issues](https://chat.zulip.org/#narrow/stream/9-issues) for issues with
-     the Zulip web app or server. A good default if you aren't sure which stream
-     to use.
+     the Zulip web app or server. This stream is a good first choice if you aren't
+     sure which stream to use.
```

Comment:
For that second sentence, how about just "Use this stream if you aren't sure which stream is most appropriate."?

Reply:
That's a definite improvement, opening with a verb. I'll put up a fix.